### PR TITLE
Widget::get_events() / add_events() / set_events() should take a gdk:…

### PR DESCRIPTION
**Pull request by sdroege**
…:EventMask

Fixes https://github.com/gtk-rs/gtk/issues/704

### CODE REVIEW

File: Gir.toml [hunk 1]
```diff
@@ -1868,6 +1868,15 @@ status = "generate"
...
         nullable = false
+    [[object.function]]
+    name = "add_events"
+    ignore = true
+    [[object.function]]
+    name = "get_events"
+    ignore = true
+    [[object.function]]
+    name = "set_events"
+    ignore = true
```
Comment (EPashkin) [inline Gir.toml]: property "events" need ignore too.

**Comment by EPashkin** [comment]
@sdroege Thanks

File: Gir.toml (hunk 1 above)
Comment (sdroege) [reply Gir.toml]: Why? One might want to use the notify signal of it

Comment (EPashkin) [reply Gir.toml]: You right, I missed that `connect_property_events_notify` already generated.
And we don't have way to generate only notify without get/set 😭

**Comment by GuillaumeGomez** [comment]
Thanks! Waiting for CIs now.

File: Gir.toml (hunk 1 above)
Comment (GuillaumeGomez) [reply Gir.toml]: @EPashkin: I wonder why too. :)

Comment (GuillaumeGomez) [reply Gir.toml]: Ah damn. Github refreshed after I commented. Now we know why. 😆

**Comment by EPashkin** [comment]
`as i32` needed in manual code

**Comment by EPashkin** [comment]
and `use gdk;`, see https://ci.appveyor.com/project/GuillaumeGomez/gtk/build/1.0.1130/job/8v2661qm45fjflnc#L232

**Comment by sdroege** [comment]
Oops, fixed!

**Comment by sdroege** [comment]
@GuillaumeGomez all green except for spurious travis failure

**Pull request merged**